p2p/peerManager/peerManager.go

```diff
@@ -122,7 +125,18 @@ func NewManager(ctx context.Context, low int, high int, datastore datastore.Data
 
 	// Initialize the expected peerDBs
 	peerDBs := make(map[string][]*peerdb.PeerDB)
-	for _, loc := range utils.GetRunningZones() {
+
+	generateLocations := func() []common.Location {
```

Comment:
we should make a ticket to track this though, so that we come back and fix this properly, but this will do for the testnet

Reply:
Created #1624 #1625